ModularTegustation/lc13_machinery.dm

```diff
@@ -0,0 +1,109 @@
+//Links to abnormality consoles when the console spawns
+/obj/machinery/containment_panel
+	name = "containment_panel"
+	desc = null
+	icon = 'ModularTegustation/Teguicons/lc13doorpanels.dmi'
+	icon_state = "frame_control"
+	density = FALSE
+	use_power = 0
+	var/department = "control"
+	var/linked_console
+	var/work
+	var/turf/consoletarget //for post spawn editing
+
+/obj/machinery/containment_panel/Initialize()
+	consoletarget = locate((x + 1), (y + 3), z) //always left of the containment door
```

Comment:
That's a little bit too hardcoded.
You should instead modify the abnormality room template and its spawning code to include it.

Reply:
Egor. Can you dictate where the machinery will be spawned and which department the abnormality will spawn in?